app/controllers/repositories_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RepositoriesController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @client      = Octokit::Client.new
+    @github_user = @client.user(current_user.uid)
+    @repos       = @client.repos(@github_user.login, query: { type: 'owner', sort: 'asc' })
```

Comment:
then after column renaming it should works simplier like this:
`@repos = @client.repos(@current_user.nickname, query: { type: 'owner', sort: 'asc' })`

Reply:
Hi, @skrix! 
Made the renaming migration and changed uid to nickname. You're right it's more explicit to understand! Nevertheless, I had to keep `user.save!` in the User's model because of this https://github.com/arkency/find-open-source-mentor/pull/15#issuecomment-529113076